pymtl3/dsl/Component.py

```diff
@@ -574,6 +574,13 @@ def get_all_update_on_edge( s ):
     except AttributeError:
       raise NotElaboratedError()
 
+  def get_all_lambda_upblks( s ):
+    try:
```

Comment:
hmm the translation pass might not need this anymore? @ptpan

Reply:
@ptpan

Reply:
Sorry for the delay. Right the translation pass now collects lambda upblks through `get_all_update_blocks`. I'm not sure if someone might be interested in getting all the lambda update blocks...

Reply:
I will be adding those back, and you might want to change how PYMTL SOURCE is dumped for those lambda blocks using the ast that I will save.

Reply:
I can hack this

Reply:
Hmm it seems like I need to pass more information from the component instance to RTLIR. The translator doesn't know the instance so it cannot call API.